pkg/controllers/awsloadbalancercontroller/credentials_request.go

```diff
@@ -36,46 +35,45 @@ func (r *AWSLoadBalancerControllerReconciler) currentCredentialsRequest(ctx cont
 	return true, cr, nil
 }
 
-func (r *AWSLoadBalancerControllerReconciler) ensureCredentialsRequest(ctx context.Context, namespace string, controller *albo.AWSLoadBalancerController) error {
-	credReq := createCredentialsRequestName(credentialRequestName)
+// ensureCredentialsRequest ensures the CredentialsRequest resource and return the secret where the credentials will be written
+func (r *AWSLoadBalancerControllerReconciler) ensureCredentialsRequest(ctx context.Context, namespace string, controller *albo.AWSLoadBalancerController) (string, error) {
+	credReq := createCredentialsRequestName(fmt.Sprintf("%s-%s", controllerResourcePrefix, controller.Name))
```

Comment:
We have 2 different variables `controllerBaseName` and  `controllerResourcePrefix`, need to reconcile to either one of them.

Reply:
I've removed the variable `controllerBaseName` for now because it was used in only one location. However there is larger refactor required where we generate the resource names in the individual ensure functions later. But that's outside the scope of this PR.